gen.go

```diff
@@ -70,6 +70,10 @@ func (s *Schema) Generate() ([]byte, error) {
 			Definition: schema,
 		}
 
+		if !s.CheckForDuplicateTitles() {
+			panic(fmt.Errorf("duplicate titles detected for %s", context.Name))
```

Comment:
Yeah, I agree. I did that initially just following how the code bailed out when an error is encountered.... will fix.